### ISSUE TITLE
how can convert labeling for pascal voc?

### ISSUE THREAD

**Issue by jeansely** [fchollet/keras#6930]
how can used labels pascal voc in python(keras).
i download dataset from ;
http://www.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/semantic_contours/benchmark.tgz

has keras  solution for this issue? or  DataProvider that provides data from a Dataset.

or a link for download , pascal voc with annotations for used in keras.

Thanks
